packages/npm2gpr/src/index.js

```diff
@@ -0,0 +1,51 @@
+const fetch = require("node-fetch");
+
+/**
+ * Get the correct url format 
```

Comment:
Add comments for each parameter as well. Similar to comments on `getSearchPage`.

Reply:
@ruwenben you missed this one